src/AppInstallerCLICore/Workflows/WorkflowBase.h

```diff
@@ -29,6 +29,17 @@ namespace AppInstaller::CLI::Workflow
         PostExecution = 5000,
     };
 
+    enum class SearchPurpose
+    {
+        Default,

```

Comment:
I feel the default is a bit confusing, default should be one of the SearchPurpose below

Reply:
Sure; I had originally included it so as to not rewrite every call to `EnsureMatchesFromSearchResult`, but that was before the multiquery PR was merged. There's still some areas where I think it makes sense, but I'll adjust the enum to remove `Default`